From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Armbian <build@armbian.com>
Date: Sat, 14 Feb 2025 00:00:00 +0000
Subject: [PATCH] ASoC: qcom: sm8250: Fix missing TDM declarations for 6.19

Patch 0005 (tdm support) may not apply cleanly on 6.19, leaving
TDM_BCLK_RATE and the ret/j variables undeclared. This adds them
so the code added by 0005/0019 compiles.

Signed-off-by: Armbian <build@armbian.com>
---
 sound/soc/qcom/sm8250.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/sound/soc/qcom/sm8250.c b/sound/soc/qcom/sm8250.c
--- a/sound/soc/qcom/sm8250.c
+++ b/sound/soc/qcom/sm8250.c
@@ -17,6 +17,8 @@
 #include "sdw.h"
 
 #define MI2S_BCLK_RATE		1536000
+#define TDM_BCLK_RATE		12288000
+
 
 struct sm8250_snd_data {
 	bool stream_prepared[AFE_PORT_MAX];
@@ -76,6 +78,7 @@ static int sm8250_snd_startup(struct snd_pcm_substream *substream)
 	struct snd_soc_pcm_runtime *rtd = snd_soc_substream_to_rtd(substream);
 	struct snd_soc_dai *cpu_dai = snd_soc_rtd_to_cpu(rtd, 0);
 	struct snd_soc_dai *codec_dai = snd_soc_rtd_to_codec(rtd, 0);
+	int ret, j;
 
 	switch (cpu_dai->id) {
 	case PRIMARY_MI2S_RX:
